pennylane_cirq/cirq_device.py

```diff
@@ -69,23 +78,31 @@ class CirqDevice(QubitDevice, abc.ABC):
     short_name = "cirq.base_device"
 
     def __init__(self, wires, shots, analytic, qubits=None):
-        super().__init__(wires, shots, analytic)
-
-        self.circuit = None
 
-        device_wires = self.map_wires(self.wires)
+        if not isinstance(wires, Iterable):
+            # interpret wires as the number of consecutive wires
+            wires = range(wires)
+        num_wires = len(wires)
```

Comment:
This should be done in ``super`` already. No harm in reproducing, but ``self.num_wires`` and ``self.wires`` should be defined at that stage! 

```suggestion
```